ts/Core/Legend/Legend.ts

```diff
@@ -1758,6 +1767,42 @@ namespace Legend {
 
     export type Item = (BubbleLegendItem|Series|Point);
 
+    /* *
+     *
+     *  Constants
+     *
+     * */
+
+    const composedMembers: Array<unknown> = [];
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+    /**
+     * @private
+     */
+    export function compose(ChartClass: typeof Chart): void {
+        if (composedMembers.indexOf(ChartClass) === -1) {
+            composedMembers.push(ChartClass);
+            addEvent(ChartClass, 'rendering', function (): void {
+                const chart = this;
+                // Title
+                chart.setTitle();
```

Comment:
This looks a bit suspicious - the chart title is not related to the legend 🤔

Reply:
I thought that it might be related to the box calculations.

Reply:
Yes, but the way it is implemented now, there will not be a chart title without the Legend module?